composer.json

```diff
@@ -18,24 +18,25 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^8.0 || ^8.1",
-        "silverstripe/vendor-plugin": "^1",
-        "silverstripe/framework": "^4.11",
-        "silverstripe/admin": "^1.11",
-        "silverstripe/userforms": "^5.14",
-        "dnadesign/silverstripe-elemental": "^4.4",
-        "silverstripe/smime": "dev-master",
-        "dnadesign/silverstripe-elemental-userforms": "^3.2",
-        "silverstripe/asset-admin": "^1.11",
-        "unclecheese/display-logic": "^2.0",
-        "lekoala/silverstripe-encrypt": "^3.4"
+        "php": "^8.1",
+        "silverstripe/vendor-plugin": "~2.0",
+        "silverstripe/framework": "~5.2",
+        "silverstripe/admin": "~2.2",
+        "silverstripe/userforms": "~6.2",
+        "dnadesign/silverstripe-elemental": "~5.2",
+        "dnadesign/silverstripe-elemental-userforms": "~4.1",
+        "silverstripe/asset-admin": "~2.2",
+        "unclecheese/display-logic": "~3.0",
+        "lekoala/silverstripe-encrypt": "~4.0",
+        "silverstripe/cms": "~5.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9",
+        "phpunit/phpunit": "~9.6",
         "squizlabs/php_codesniffer": "^3.0",
         "php-parallel-lint/php-parallel-lint": "^1.2",
         "slevomat/coding-standard": "~6.0",
-        "nikic/php-parser": "^4.15"
+        "nikic/php-parser": "^4.15",
+        "silverstripe/recipe-testing": "~3.0"
```

Comment:
Boooooy this was a pain!! 

This is what stopped Github actions from working as it kept attempting to look for classes from [silverstripe-behat-extension](https://github.com/tractorcow/silverstripe-behat-extension/tree/master). 

According to Guy S. there may be a bigger issue in play here with [gha-ci](https://github.com/silverstripe/gha-ci) in that its configs are pulling in `behat` related tests when we are in the context of `unit` tests. Ideally, these should be isolated and shouldn't pull in files from a different suite. 

This will be noted in our SS5 gotchas page since Guy has suggested not to raise it given the low effort workaround.